openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml

```diff
@@ -0,0 +1,21 @@
+- name: "A single parent with a child situation"
+  description: A simple situation just to test
```

Comment:
As a best practice example, we should be as clear and literate as possible (or not add a `description` at all). Maybe this test is about “Income tax should get properly computed across all members of a household”? 😃